### ISSUE TITLE
Update vendor libraries

### ISSUE THREAD

**Issue by skurvish** [joomlahenk/fabrik#34]
Many of the vendor libraries used by the current version are outdated. We may or may not wish to continue support for the functionality but I want to list the ones I come across so we can make decisions along the way.

dompdf
Twilio
Mailgun
AWS (Amazon web service)
Myclabs
phenx

**Comment by trob** [comment]
domPDF: 
https://fabrikar.com/forums/index.php?threads/dompdf-and-php8.53405/ 
fstorch: "...My PDF looks good again with PHP8 and DOMPDF 2.0. "
I'll test it.

**Comment by skurvish** [comment]
Yes, I was aware of that one. I think we need to update the composer.json file properly rather than simply ignoring the dependencies in it.

**Comment by joomlahenk** [comment]
Do we realy need all these vendor libraries b.t.w. Or some only for a plugin (which could be installed together with the plugin).
My feeling is that we have far to many vendor libraries and most are never used.

**Comment by trob** [comment]
`I read somewhere in the fabrik code that DOMPDF only can be used if DOMPDF is installed/present in Joomla`No, it was always only Fabrik. Fabrik is checking if a folder exists because the PDF libs are not part of Fabrik core and you have to 'install' the library package. But it's not really installing, it's just files copying, you can also just update from GitHub without doing a discover.

`Do we realy need all these vendor libraries` no idea. We can try to rename the folders and see what happens ;)

**Comment by trob** [comment]
composer and symfony are also in Joomla's vendor (identical???),
I assume stripe and twilio are for the plugins

**Comment by trob** [comment]
@skurvish : where do you see Mailgun and zoom?

**Comment by skurvish** [comment]
One option I have been thinking about is integrating [Koolreport](https://www.koolreport.com/) into Fabrik (I already use it front ended by Fabrik forms). It has the ability to do a whole bunch of things that the visualizations do as well as output to pdf. The actual pdf generation is done offsite through an ajax call so there is no code burden on Fabrik for this. Something to ponder for future consideration.

**Comment by joomlahenk** [comment]
Removal of DOMPDF / MPDF choise in the config.xml undone.

**Comment by trob** [comment]
domPDF is updated

**Comment by joomlahenk** [comment]
That is, I still used to old 0.8.6 version. Updated with GH now.
